app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -142,6 +142,13 @@ def scsi_controller_types
       }
     end
 
+    def firmware_types
+      {
+        "bios" => N_("BIOS"),
+        "efi" => N_("EFI")
```

Comment:
Wikipedia says that UEFI is the standard now, I've decided to go with UEFI everywhere in Foreman core for now.